mmdet/core/evaluation/text_evaluation.py

```diff
@@ -43,9 +43,10 @@ def draw_gt_polygons(image, gt_polygons, gt_dont_care_nums):
 
     for point_idx, polygon in enumerate(gt_polygons):
         color = (128, 128, 128) if point_idx in gt_dont_care_nums else (255, 0, 0)
-        for i in range(4):
+        plen = len(polygon[0])
```

Comment:
tlen

Reply:
Don't you like the name? Should I change it?

Reply:
No, it's all good.